lib/engine/game/g_1848/game.rb

```diff
@@ -142,37 +142,43 @@ class Game < Game::Base
                     train_limit: 4,
                     tiles: %i[yellow green],
                     operating_rounds: 2,
-                    status: ['can_buy_companies'],
+                    status: %w[can_buy_companies can_buy_trains],
                   },
                   {
                     name: '4',
                     on: '4',
                     train_limit: 3,
                     tiles: %i[yellow green],
                     operating_rounds: 2,
-                    status: ['can_buy_companies'],
+                    status: %w[can_buy_companies can_buy_trains],
```

Comment:
Recommend making can_buy_train an event, since the effect lasts the rest of the game.

Reply:
done